src/vswhere.lib/JsonFormatter.cpp

```diff
@@ -7,91 +7,91 @@
 
 using namespace std;
 
-void JsonFormatter::StartArray(_In_ wostream& out)
+void JsonFormatter::StartArray(_In_ Console& console)
 {
     m_arrayStart = true;
 
-    out << m_padding << L"[";
+    console.Write(L"%ls[", m_padding.c_str());
     Push();
 }
 
-void JsonFormatter::StartObject(_In_ wostream& out)
+void JsonFormatter::StartObject(_In_ Console& console)
 {
     m_objectStart = true;
 
     if (m_arrayStart)
     {
-        out << endl;
+        console.WriteLine();
     }
     else
     {
-        out << L"," << endl;
+        console.Write(L",\n");
```

Comment:
That would create a std::setting, though I could add an overload to avoid that. Starting to try and decrease memory pressure.